apps/examples/testcase/ta_tc/wifi_manager/utc/Kconfig

```diff
@@ -0,0 +1,95 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC
+	bool "Wi-Fi Manager UTC TestCase Example"
+	select WIFI_MANAGER
+	default n
+	---help---
+		Enable the Wi-Fi Manager TestCase Example
+
+if EXAMPLES_TESTCASE_WIFI_MANAGER_UTC
+comment "Specify your AP which you want to connect to"
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SSID
+    string "SSID of AP"
+    default "NULL"
+    ---help---
+		Select SSID of AP which you want to connect to
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SSID_LEN
+    int "String length of SSID"
+    default 0
+    ---help---
+		Specify the length of SSID
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_PASSPHRASE
+    string "Passphrase of AP"
+    default "NULL"
+    ---help---
+		Passphrase of AP which you want to connect to
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_PASSPHRASE_LEN
+    int "String length of passphrase"
+    default 0
+    ---help---
+		Specify the length of passphrase
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_AUTHENTICATION
+    int "Authentication type"
+    default 0
+    ---help---
+		0 : Open
+		1 : WEP Shared
+		2 : WPA PSK
+		3 : WPA2 PSK
+		4 : WPA and WPA2 PSK
+		5 : Unknown
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_CRYPTO
+    int "Encryption type"
+    default 0
+    ---help---
+		0 : None
+		1 : WEP 64 bits
+		2 : WEP 128 bits
+		3 : AES
+		4 : TKIP
+		5 : TIKP and AES
+		6 : Unkown
+
+comment "Specify SoftAP which is used for easy-setup" 
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SOFTAP_SSID
+    string "SSID of SoftAP"
+    default "PKES"
+    ---help---
+		Name SSID of SoftAP
+
+config EXAMPLES_TESTCASE_WIFI_MANAGER_UTC_SOFTAP_SSID_LEN
+    int "String length of SSID of SoftAP"
+    default 4
```

Comment:
is it needed? I think it could calculate SSID length from SSID

Reply:
I remove it